common/app/Flash/flash.less

```diff
@@ -13,3 +13,7 @@
   flex: 1 0 0px;
   color: #37474f;
 }
+
+#@{ns}-board {
+  margin-top: 50px;
+}
```

Comment:
Add newline when ending a file, or configure your editor for it.

Reply:
@raisedadead Done.